src/ui/ban_modal.py

```diff
@@ -0,0 +1,31 @@
+import discord
+from services.ban_service import ban_user
+from .helper import create_modal_embed
+
+
+class BanModal(discord.ui.Modal):
+
+    def __init__(self, user: discord.Member) -> None:
+        self.user = user
+        super().__init__(title=f"Ban User {user.display_name}")
+
+    reason = discord.ui.TextInput(
+        label="Ban Reason",
+        style=discord.TextStyle.long,
+        placeholder="Why is the user banned",
```

Comment:
i think we can change this to "Reason for banning user"

Reply:
Agreed, done